Gruntfile.js

```diff
@@ -135,6 +135,8 @@ module.exports = function (grunt) {
                             '!thirdparty/CodeMirror2/mode/**/*.html',
                             '!thirdparty/CodeMirror2/**/*test.js',
                             'thirdparty/CodeMirror2/theme/{,*/}*',
+                            'thirdparty/filer-dialogs/bower_components/font-awesome/css/font-awesome.min.css ',
```

Comment:
It looks like HTMLHinter isn't using font-awesome. Is this still worth changing?